src/CoreExtensions/ClassDefinition/IndexFieldSelectionCombo.php

```diff
@@ -25,13 +25,6 @@
 
 class IndexFieldSelectionCombo extends Select
 {
-    /**
-     * Static type of this element
-     *
-     * @var string
-     */
-    public string $fieldtype = 'indexFieldSelectionCombo';
```

Comment:
BC break?

Reply:
The attribute still exists in the parent class. See https://github.com/pimcore/pimcore/pull/15770
I would deprecate it, it shouldn't used anymore.

Do you want to keep this overwrite and deprecate it here also?

Reply:
Add it again with deprecation:
https://github.com/pimcore/ecommerce-framework-bundle/pull/119/commits/ddf56f580be9cb54f431b0c3c94cc2020b892d60